release/releaser.go

```diff
@@ -203,8 +203,10 @@ func logEvent(inst *instance.Instance, executeErr error, release flux.Release) e
 	}
 
 	var serviceIDs []flux.ServiceID
-	for _, id := range release.Result.ServiceIDs() {
-		serviceIDs = append(serviceIDs, flux.ServiceID(id))
+	for k, v := range release.Result {
```

Comment:
I initially had that in, but then took it out, because the previous functionality was to include skipped services in the history message.

If you think we should be ignoring skipped services too, let me know and I'll change.